app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/LoadFilesListTask.java

```diff
@@ -111,6 +111,13 @@ public LoadFilesListTask(
 
     nullCheckOrInterrupt(mainFragment, this).folder_count = 0;
     nullCheckOrInterrupt(mainFragment, this).file_count = 0;
+    int numericPath = -1;
+    try {
+      numericPath = Integer.parseInt(path);
```

Comment:
I don' think we should do so much of recomputation just to fix a small issue. You're almost always calculating openmode here again on each load list task, moreover this'll fill logs with warn messages.
I think this try/catch should be moved to switch case `CUSTOM` and maybe you can try to load list again if possible otherwise show a way to denote a failed operation (unsupported operation toast or something similar).